service-base/src/main/java/fi/nls/oskari/util/IOHelper.java

```diff
@@ -828,4 +859,47 @@ public static String getParams(Map<String, String> params) {
         // drop last character ('?' or '&')
         return urlBuilder.substring(0, urlBuilder.length()-1);
     }
+
+    /**
+     * Very similiar to IOHelper.getParams(Map<String, String> params)
+     * only difference is that this method encodes the keys as well as the values
+     * @param kvps map containing the key value pairs
+     * @return String
+     */
+    public static String formURLEncode(final Map<String, String> kvps) {
```

Comment:
Should getParams() implementation just use this one so we would not have a almost identical functions with developers not sure which one to use?